clients/lsp-wgsl.el

```diff
@@ -101,11 +101,11 @@
   :group 'lsp-wgsl
   :package-version '(lsp-mode . "8.0.1"))
 
-;; (defcustom lsp-wgsl-shaderdefs []
-;;   "Defines that should be valid for preprocessor operations like ifdef."
-;;   :type 'vector
-;;   :group 'lsp-wgsl
-;;   :package-version '(lsp-mode . "8.0.1"))
+(defcustom lsp-wgsl-shaderdefs []
+  "Defines that should be valid for preprocessor operations like ifdef, e.g, ['USE_TYPES', 'DEBUG']"
+  :type '(repeat string)
```

Comment:
it should be lsp-string-vector

Reply:
Pushed now 🙂 There are many other clients (e.g, xml, erlang) that uses `(repeat string)`, so I guess those should be changed in the future as well... (aka technical debt)?